pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -66,6 +66,22 @@ var appPersistPaths = []string{
 	types.AppImgDirname,
 }
 
+func encodeErrorInfo(errStr string, errTime time.Time) *info.ErrorInfo {
```

Comment:
You might want to consider passing ErrorAndTime here, once the other PR makes it in.

Reply:
Yeah.. Need to be updated once u r PR merges up..

Reply:
Done..